maze_dataset/tokenization/maze_tokenizer.py

```diff
@@ -539,14 +903,98 @@ class Coords(_AdjListTokenizer):
         Represents a connection as the tokens of 2 coords with optional delimiters.
 
         # Parameters
-        - `intra`: Whether all coords include a delimiter token between coordinates
-        - `post`: Whether all coords include an integral following delimiter token
-        - `walls`: Whether the tokenized adjacency list should list the walls in the maze rather than the connections.
+        - `pre`: Whether all edge groupings include a preceding delimiter token
+        - `post`: Whether all edge groupings include a following delimiter token
+        - `shuffle_d0`: Specifies how to sequence the edge groupings.
+        If true, groupings are shuffled randomly. If false, groupings are sorted by the leading coord of each group.
+        - `edge_grouping`: Specifies if/how multiple coord-coord connections are grouped together in a token subsequence called a edge grouping.
+        - `edge_subset`: Specifies the subset of lattice edges to be tokenized.
+        - `edge_permuter`: Specifies, in each edge tokenization, which coord either:
+          1. Appears first in the tokenization, for `AdjListCoord`.
+          2. Is tokenized directly as a coord, for `AdjListCardinal`.
+          - `shuffle`: For each edge, the leading coord is selected randomly.
+          - `all`: Each edge appears twice in the tokenization, appearing with both leading coords.
+          - `evens`, `odds`: The leading coord is the one belonging to that coord subset. See `EdgeSubsets.ChessboardSublattice` for details.
         """
 
-        intra: bool = serializable_field(default=True)
+        pre: bool = serializable_field(default=False)
```

Comment:
To avoid what I see as unnecessary complexity, every delimiter option is just a bool. When set to `True`, each option adds a unique delimiter to the tokenization not used anywhere else. So there's no way with this design to use the same delimiter in the adjacency list as the path, but I think that's an acceptable limitation for the gains in simplicity.